jbpm-wb-process-runtime/jbpm-wb-process-runtime-api/src/main/java/org/jbpm/workbench/pr/model/DocumentSummary.java

```diff
@@ -24,6 +24,8 @@
 public class DocumentSummary extends GenericSummary<String> {
 
     private String documentId;
+    private String documentServerTemplateId;
+    private String documentIdentifier;
```

Comment:
as mentioned in the other comment, I don't think we need these two fields at all...